build/steps-selfcontained.yml

```
@@ -0,0 +1,45 @@
+##
+# Pack IQ# self-contained executable.
+##
+
+steps:
+
+##
+# Pre-reqs
+##
+- task: DownloadBuildArtifacts@0
+  inputs:
+    artifactName: iqsharp
+    downloadPath: '$(System.DefaultWorkingDirectory)/artifact'
+
+- pwsh: |
+    Move-Item "$Env:SYSTEM_DEFAULTWORKINGDIRECTORY/artifact/iqsharp/drops" "$Env:SYSTEM_DEFAULTWORKINGDIRECTORY/drops"
+  displayName: "Copy build artifacts to IQ# working directory"
+
+- task: CondaEnvironment@1
+  inputs:
+    packageSpecs: "python=3.7 pip setuptools pytest jupyter"
+  displayName: 'Use conda environment w/ Python 3.'
+
+- task: UseDotNet@2
+  displayName: 'Use .NET Core SDK 3.1.100'
+  inputs:
+    packageType: sdk
+    version: '3.1.100'
+
+##
+# Pack
+##
+- pwsh: .\pack-selfcontained.ps1
```

This shouldn't matter on `pwsh`, but suggesting for consistency with other changes in this PR.
```suggestion
- pwsh: ./pack-selfcontained.ps1
```